sources/functions/kepubify

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
+
+_kepubify() {
+    whereami=$(dirname "$0") && whereami="${whereami##*/}"
+
+    case "${whereami}" in
+        "install")
+            kepubify_action="Installing"
+            ;;
+        "upgrade")
+            if [[ -f /install/.calibreweb.lock ]]; then
+                kepubify_action="Upgrading"
+            else
+                return
+            fi
+            ;;
+        "remove")
+            kepubify_action="Removing"
+            ;;
+        *)
+            echo_error "The directory origin ${whereami} is not supported by this function"
+            exit 1
+            ;;
+    esac
+
+    echo_progress_start "${kepubify_action} kepubify apps"
+
+    case "$(_os_arch)" in
+        "amd64")
+            kepubify_arch="64bit"
+            ;;
+        "arm64")
+            kepubify_arch="arm64"
+            ;;
+        "armhf")
+            kepubify_arch="arm"
+            ;;
+        *)
+            echo_error "$(_os_arch) not supported by this app"
+            exit 1
+            ;;
+    esac
+
+    kepubify_url="https://github.com/pgaskin/kepubify/releases/latest/download"
```

Comment:
Can we move this to use the new `github_latest_version` instead before merge? I think that now that we have this as a function we should just use that.

Reply:
Probably, do the thing ;)